src/artemis/experiment_plans/full_grid_scan.py

```diff
@@ -76,7 +78,9 @@ def detect_grid_and_do_gridscan(
 ):
     # Start stage with asynchronous arming here
     yield from bps.abs_set(eiger.do_arm, 1, group="arming")
-
+    yield from bps.abs_set(
+        attenuator, parameters.artemis_params.ispyb_params.transmission
+    )
```

Comment:
Must: We need to wait on this to be done in the gridscan. In fact it falls under the same sort of wait as `arming`, I think we rename arming (here and in the wait) to something like "ready_for_data_collection" then put the `set_transmission` in the same group